pkg/cmd/playback.go

```diff
@@ -263,6 +263,7 @@ func runListen(cmd *cobra.Command, address string, wg *sync.WaitGroup) error {
 		SkipVerify:            false,
 		Log:                   logger,
 		NoWSS:                 false,
+		Timeout:               30,
```

Comment:
doesn't seem like there's a need to add the private flag to here actually? it seems like there's only one call to `proxy.init` and most paramaters are getting passed default values